server/src/test/scala/org/apache/livy/server/interactive/SessionHeartbeatSpec.scala

```diff
@@ -80,7 +80,7 @@ class SessionHeartbeatSpec extends FunSpec with Matchers {
       n.register(nonExpiredSession)
       n.deleteExpiredSessions()
 
-      verify(expiredSession).stop()
+      verify(expiredSession, atLeastOnce).stop()
```

Comment:
Is this to make it explicit? default is atLeastOnce I believe, but wondering why making it explicit here?
https://static.javadoc.io/org.mockito/mockito-core/1.9.5/org/mockito/Mockito.html#4

Reply:
default is one, but I think due to timing issues it could be called more than once, I should probably remove this change, it is unrelated, and I just tripped over it when trying to run all the existing tests and this one was failing ... the tests are generally rather unstable it seems